### PR TITLE
Shift test numeration and add gym parsing

### PR DESCRIPTION
**Pull request by julianferres**
Hi! I consider that test should start in 1 (to be more user friendly). Also added gym parsing, since it was only needed to add a parameter

### CODE REVIEW

File: cfparser.py [hunk 1]
```diff
@@ -112,13 +111,14 @@ def download_cf_contest(contest, root_path, templates_dir=None):
...
+    problem_list = parse_contest_for_problem_list(contest_type, contest)
     contest_root = os.path.join(root_path, str(contest))
 
     for problem in problem_list:
         logger.info('Parsing problem %s', problem)
-        tests_data = parse_problem(contest, problem)
-        tests_path = os.path.join(contest_root, problem, TESTS_SUBDIR)
+        tests_data = parse_problem(contest_type, contest, problem)
+        tests_path = os.path.join(contest_root, problem)
+        # , TESTS_SUBDIR)
```
Comment (mratkovic) [inline cfparser.py]: this is leftover from original code, you can remove this line

**Review by mratkovic — CHANGES_REQUESTED** [review]
Awesome, thanks for the update 👍 

rest of the PR looks good to me

**Comment by mratkovic** [comment]
@julianferres, also please update examples and usage sections in README.md with added contest type argument

**Comment by julianferres** [comment]
Done

**Review by mratkovic — APPROVED** [review]

**Pull request merged**